src/main/java/org/cp/confisual/nevisauth/NevisAuthConfigVisitor.java

```diff
@@ -7,4 +7,5 @@
 public interface NevisAuthConfigVisitor {
 	boolean visit(AuthState authState);
 	void visit(Domain domain);
+	String getSourceBuilder();
```

Comment:
Why I don't see the update in implementation class while new method is added to the interface?